### DISCUSSION TITLE
윈도우 터미널 관련 질문 있습니다!

### DISCUSSION THREAD

**Discussion by tlsdnr1135** [tedilabs/fastcampus-devops#80] [Q&A] [answered]
1. 윈도우 터미널에서 이렇게 오류가 뜹니다. 구글링 해봐도 오류가 안나오네요
![오류 1](https://user-images.githubusercontent.com/50544205/184711057-dabaaa1d-9cc3-4eed-829e-de0a0757316c.PNG)
![오류 2](https://user-images.githubusercontent.com/50544205/184711165-6640e7e6-c8f9-4166-99d3-108e8d2cbe86.PNG)

제가 생각하기에는 윈도우 터미널이 Ubuntu 2004를 못찾는거 같은데.. 
어떻게 해결해야하나요?

2. WSL은 잘 깔린거같은데, 위에 사진처럼 윈도우 터미널에서 하지 않고 그냥 Ubuntu 20.04 LTS 를 앱으로 켜서 해도 똑같은 건가요?
(그렇다면 당분간은 따로 켜서 하면 될거같아서요)

3. 밑에 사진에 cmd 에서 wsl 명령어를 치는데, 이걸 하는건 단순히 잘 실행되나 확인하기 위해서 인가요?
아니면 wsl 명령어를 치고나면 Ubuntu LTS를 사용하는거랑 똑같은 효과가 되는건가요?
![오류 2](https://user-images.githubusercontent.com/50544205/184712672-3eb0214a-698e-4d10-9111-db5e62e85552.PNG)

**Comment by bluetsys** [comment, marked as answer]
windows terminal에 보시면 밑에 화살표를 누르면 설정이란 부분이 있습니다. 설정을 눌러서 ctrl+, 에서 프로필에 찾아 보시면 Ubuntu항목을 찾으셔서 명령줄 항목을 수정 해주시면 됩니다.
C:\WINDOWS\system32\wsl.exe -d [WSL 이름] 예
C:\WINDOWS\system32\wsl.exe -d Ubuntu-20.04
